package.json

```diff
@@ -33,14 +33,14 @@
     "eslint-config-lob": "^2.2.0",
     "generate-changelog": "^1.0.2",
     "hapi": "^13.4.0",
+    "inject-then": "^2.0.6",
     "istanbul": "^0.4.4",
     "mocha": "^2.5.3",
-    "redis": "^2.6.2",
-    "redis-commands": "^1.2.0",
-    "then-redis": "^2.0.1"
+    "redis": "^0.12.1",
```

Comment:
Was there any issue with using the latest version of Redis? I'm a bit scared of a pre-v1 version, but it should be fine since we've already been using it. Did you see if these worked with redis v2.6.2?